lib/middleware/authentication.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Authentication module composed of an Express middleware used to validate
+ * incoming requests from the Slack API for Slash commands and outgoing
+ * webhooks.
+ */
+
+// the list of authorized tokens that can invoke slash comamnds
+const TOKEN_NOT_FOUND = -1;
```

Comment:
Can you replace `const` with `var` please? We're trying to support NodeJS 0.12.x also.